ICRC-1.did

```diff
@@ -39,7 +43,8 @@ service : {
     icrc1_symbol : () -> (text) query;
     icrc1_decimals : () -> (nat8) query;
     icrc1_total_supply : () -> (nat) query;
-    icrc1_balance_of : (record { of: principal; subaccount: opt SubAccount; }) -> (nat) query;
-    icrc1_transfer : (TransferArgs) -> (variant { Ok: nat; Err: TransferError; });
+    ircr1_minting_acount : () -> (Account) query;
```

Comment:
Typo: `ircr1_minting_acount` should be `icrc1_minting_acount

Reply:
Thanks for catching!